src/ui/public/vis/agg_configs.js

```diff
@@ -9,9 +9,10 @@
 
 import _ from 'lodash';
 import { IndexedArray } from 'ui/indexed_array';
-import VisAggConfigProvider from 'ui/vis/agg_config';
+import { VisAggConfigProvider } from 'ui/vis/agg_config';
 import AggTypesIndexProvider from 'ui/agg_types/index';
```

Comment:
Looks like this file got left out of the a - m chunk.  I'll add the conversion in here. It still works bc I kept the default export around in that file.